extensions/mongodb/src/main/java/com/hazelcast/jet/mongodb/impl/MongoUtilities.java

```diff
@@ -167,4 +167,14 @@ static void checkDatabaseExists(MongoClient client, String databaseName) {
         ClusterDescription clusterDescription = client.getClusterDescription();
         throw new JetException("Database " + databaseName + " does not exist in cluster " + clusterDescription);
     }
+
+
+    public static void checkDatabaseAndCollectionExists(MongoClient client, String databaseName, String collectionName) {
+        checkDatabaseExists(client, databaseName);
+        MongoDatabase database = client.getDatabase(databaseName);
+        if (collectionName != null) {
+            checkCollectionExists(database, collectionName);
+        }
+    }
```

Comment:
nit: I would put this method above the two methods above, it then reads better.

Reply:
d365f2f8795dc76b6ecdab7884db53e094f7e06a